lib/linguist/languages.yml

```diff
@@ -8590,6 +8590,17 @@ sed:
 templ:
   type: markup
   color: "#66D0DD"
+varlink:
+  type: programming
```

Comment:
Is this really a programming language. Feels more like configuration so would fall under data

Reply:
I think it's a markup language as a varlink file describes interfaces (see https://en.wikipedia.org/wiki/Interface_description_language). What do you think?

Reply:
Markup is for things that result in visual effects, think html. From what I've read, the only varlink files that will be committed are the interface definition files which are data by the fact they're definition files… they're defining the methods the interface implements.